schema/annotation.yaml

```diff
@@ -228,7 +228,13 @@ classes:
     exact_mappings:
       - biolink:GeneProduct
 
+  annotation:
+    slots:
+      - type
+      - was generated by
+        
   functional annotation:
+    is_a: annotation
```

Comment:
discussed this with @wdduncan - we think adding a new superclass is overkill here